File: Palindrome Number/palindrome-number.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        elif 0 <= x < 9:
            return True

        num = x
        reverse = ""
        while num > 0:
            reverse += str(num % 10)
            num //= 10

        return x == int(reverse)


solution = Solution()
print(solution.isPalindrome(121))  # True

File: Palindrome Number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
const isPalindrome = function(x) {
  if (x < 0) {
    return false;
  } else if (x >= 0 && x < 10) {
    return true;
  }

  let num = x;
  let reverse = "";

  while (num > 0) {
    reverse += num % 10;
    num = Math.floor(num / 10);
  }

  return x == reverse;
};

console.log(isPalindrome(123)); // false
console.log(isPalindrome(121)); // true